cli/src/main.rs

```diff
@@ -22,6 +22,10 @@ struct Arg {
 
 #[derive(Subcommand)]
 enum Command {
+    /// Contains subcommands related to admin.
```

Comment:
```suggestion
    /// Contains subcommands related to server and organization admin.
```

To make clear that some organization commands are here, not in the organization group